python/GafferDispatchTest/DebugDispatcher.py

```diff
@@ -109,7 +109,7 @@ def __acquireNode( self, batch, dispatchData ) :
 	def __createNode( name ) :
 
 		node = Gaffer.Node()
-		node.setName( name )
+		node.setName( name.replace( ".", "::" ) )
```

Comment:
It seems like this would trigger the invalid name exception in [GraphComponent::setName()](https://github.com/GafferHQ/gaffer/blob/master/src/Gaffer/GraphComponent.cpp#L80)? I think '_' is probably the most sensible replacement character...